apps/categories2/stylesheets/index.styl

```diff
@@ -0,0 +1,4 @@
+.categories-page
+  padding 40px 0
+
+.categories-page__header
```

Comment:
Again, fine b/c it's just boilerplate but heads up I believe not specifying any rules will make Stylus choke.